### DISCUSSION TITLE
How to change the friction coefficient and elasticity coefficient of a rectangle arena in Webots R2021a?

### DISCUSSION THREAD

**Discussion by gh18406468101** [cyberbotics/webots#6384] [General]
<img width="241" alt="image" src="https://github.com/cyberbotics/webots/assets/107991095/58aa0977-32d9-4700-8c6d-8e89916e421f">

I can't find the options regarding friction coefficient and elasticity coefficient. How should I change them, or can I say that this cannot be changed

**Comment by ShuffleWire** [comment]
It possible : 
You have to setup contactProperties in the WorldInfo 
![image](https://github.com/cyberbotics/webots/assets/83014015/9cf3d0b0-1651-4118-bc34-43c1c77f0457)
If not changed, the material of your arena is "default"
![image](https://github.com/cyberbotics/webots/assets/83014015/ecd47383-2da4-4411-bc84-3c8e5d3644c5)
and the name of the material2 should be the name of the object colliding it
I let you read
https://cyberbotics.com/doc/reference/contactproperties
and play with
asymmetric_frinction1 and asymmetric_frinction2 (you can find them in the sample list)
